### ISSUE TITLE
Import from Evernote

### ISSUE THREAD

**Issue by orthecreedence** [turtl/project-tracker#22]
This needs a parser in the core, and a small update in the UI.

It shouldn't be too hard to parse the import (I believe it's just an XML file).

**Comment by magnusmanske** [comment]
Hi, I have written [a small stand-alone tool](https://github.com/magnusmanske/evernote2turtl) in Rust that can convert a local Evernote export (ZIP file or unzipped directory with one html file per note) into a turtl JSON backup file, which can then be imported into turtl.

I tried it on my own Evernote export, and it basically works. All notes show up in a new "Evernote import" space after import; board information seems to get lost on Evernote export. Things like checkbox elements don't get imported correctly, and information from Evernote such as location (of first edit?) that is stored in the export files gets dropped. Also, no file import, though should be possible in principle.

Caveat: This is literally my first ever Rust program, so it probably looks horrible to the trained eye, but it works. Feel free to copy/expand the module into your code.

**Comment by orthecreedence** [comment]
@magnusmanske This is really cool! Thanks for the contribution. Would you mind [signing the CLA](https://turtlapp.com/contributing/icla/) so we can include this code in the project?

This is a great starting point, and much appreciated.

**Comment by magnusmanske** [comment]
I have signed this, even though I'm not sure why that was necessary, since we both use GPLv3...

**Comment by magnusmanske** [comment]
I have added Google Keep import (for unzipped directory only), thereby also solving #161 .
